grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/RequestContext.java

```diff
@@ -138,4 +138,15 @@ public ContextualKey<Void> buildInternalContextualKey() {
   public <T> ContextualKey<T> buildInternalContextualKey(T data) {
     return new DefaultContextualKey<>(this, data, List.of(TENANT_ID_HEADER_KEY));
   }
+
+  @Override
```

Comment:
nit: we typically use lombok so it gets updated automatically

Reply:
but doesn't lombok use the class members to generate the string? this is kind of specific to what we want - not the class members as such..

Reply:
It's configurable. Can mark getter methods if you want I believe (and most of the jwt details are already exposed as getters)

Reply:
oh saw ur msg after merging - I guess should be ok for now...